### PR TITLE
Adding AWS S3 Example

### PR DESCRIPTION
**Pull request by nniehoff**
This PR provides a brief example of how someone might host static files in AWS S3.  This is intended to be a supplement to #673.

### CODE REVIEW

File: examples/s3_static_files/README.md
Comment (glennmatthews) [inline examples/s3_static_files/README.md:7]:
```suggestion
Nautobot can use [`django-storages`](https://django-storages.readthedocs.io/en/stable/) to publish files to S3 -- see the [docs](https://nautobot.readthedocs.io/en/stable/installation/nautobot/#remote-file-storage) for more information.
```

File: examples/s3_static_files/README.md
Comment (glennmatthews) [inline examples/s3_static_files/README.md:42]:
```suggestion
The AWS S3 bucket will be hosting Nautobot static files and needs some specific configuration to allow anonymous HTTP access.  The following is the necessary Terraform configuration to create the S3 bucket appropriately:
```

File: examples/s3_static_files/README.md
Comment (glennmatthews) [inline examples/s3_static_files/README.md:38]: "internally looks up IAM credentials..." from where?

```suggestion
If `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` are not set, `boto3` internally looks up IAM credentials.  Using an [IAM Role for EC2](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html?icmpid=docs_ec2_console) is highly recommended.
```

**Review by glennmatthews — COMMENTED** [review]
Thank you for the pull request!

File: examples/s3_static_files/README.md
Comment (FragmentedPacket) [inline examples/s3_static_files/README.md:33]: Could we potentially just do:

```
    "USE_TZ": USE_TZ,
```

It looks like the nautobot_config.py imports in the settings. Not sure if the potential for someone to set the global `USE_TZ` to something different than the `USE_TZ` stored within `STORAGE_CONFIG`.

**Review by FragmentedPacket — CHANGES_REQUESTED** [review]

File: examples/s3_static_files/README.md
Comment (nniehoff) [reply examples/s3_static_files/README.md:33]: I think I'd either rather set it explicitly to `True` here or wait until PR #673 is merged and just remove it here

**Review by FragmentedPacket — APPROVED** [review]

File: examples/s3_static_files/README.md
Comment (FragmentedPacket) [reply examples/s3_static_files/README.md:33]: That works. I'm not sure if #673 will make it into 1.1.0 or if we're going to get rid of the monkey patching completely in a later release.

File: examples/s3_static_files/README.md
Comment (jathanism) [inline examples/s3_static_files/README.md:41]: The mention of Terraform here appears to be non-sequitur? Is this specific to how we are using S3 with Nautobot?

Comment (nniehoff) [reply examples/s3_static_files/README.md:41]: It's provided as an example of how it might be done.  Documenting the manual process of setting the correct bucket settings involves a lot of steps

Comment (glennmatthews) [reply examples/s3_static_files/README.md:41]: Perhaps change the wording to something like "The following is an example of Terraform configuration to create the S3 bucket appropriately"?

**Review by glennmatthews — APPROVED** [review]
Thank you!

**Pull request merged**
